bash_completion.sh

```diff
@@ -237,7 +237,7 @@ _buildtest ()
     ;;
 
     config|cg)
-      local cmds="-h --help co compilers e edit ex executors p path systems validate v view"
+      local cmds="-h --help co compilers e edit ex executors p path systems validate v view profiles"
```

Comment:
`profiles` should be `profile`. This is what you wrote in buildtest/cli/config.py